types/greasemonkey/index.d.ts

```diff
@@ -105,6 +105,8 @@ declare namespace GM {
         user?: string | undefined;
         /** Password to use for authentication purposes */
         password?: string | undefined;
+        /** Decode the response as specified type. Default value is "text" */
+        responseType?: XMLHttpRequestResponseType | undefined;
```

Comment:
Maybe we don't need `| undefined` if value is already optional `?:`

Reply:
I'm not sure about this, other properties all have `| undefined` so I add it.